src/gen_rpc.app.src

```diff
@@ -5,25 +5,26 @@
 %%% Copyright 2021 Miniclip. All Rights Reserved.
 %%%
 
-{application, gen_rpc,
-    [{description, "A scalable RPC library for Erlang-VM based languages"},
+{application, gen_rpc, [
+    {description, "A scalable RPC library for Erlang-VM based languages"},
     {vsn, git},
+    {registered, []},
     {mod, {gen_rpc_app, []}},
-    {registered, [gen_rpc_dispatcher]},
-    {maintainers, ["Miniclip"]},
-    {licenses, ["Apache 2.0"]},
+    {pkg_name, mc_gen_rpc},
+    {licenses, ["Apache License, Version 2.0"]},
     {links, [{"Github", "https://github.com/miniclip/gen_rpc"}]},
-    {files, [
```

Comment:
Not required.

Reply:
Hm... this seems to be actually required for Hex publishing to be trimmed. Oh well.

Reply:
That sounds unusual, I don't think I've ever had to use it.

Unless you have code / `rebar` artifacts placed in non-standard folders, the defaults should work. (And because it doesn't, I'd say something else is out of place)